pkg/sql/codegen/tensorflow/template_analyze.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tensorflow
+
+import "sqlflow.org/sqlflow/pkg/sql/ir"
+
+type analyzeFiller struct {
```

Comment:
How about renaming analyzeFiller into explainFiller?

Reply:
I'll fix it in the next PR.